docs/_documentations/remotedeploy-projects-eclipse.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: docs
+title: Eclipse
+description: Creating and importing projects
+keywords: users, projects, Kubernetes, LDAP, user management, access management, login, deployment, pod, security, securing cloud connection, remote deployment of Codewind
+duration: 5 minutes
+permalink: remotedeploy-projects-eclipse
+type: document
+---
+
+# Creating and importing projects
+
+When you configure Codewind to be used remotely, you are ready to create a project that you can build and run remotely. 
+
+If you have projects that currently exist on your local computer that you would like to continue to develop remotely, follow [Import an existing local project to the cloud](##import-an-existing-local-project-to-the-cloud).
+
+## Create a new project
+
+Go to the Codewind pane and click the Plus (+) icon on the new cloud deployment:
+
+![Creating new remote project](./images/remotedeploy-projects-eclipse/eclipse_create-new-project.png){:width="380"}
+
+Select the template type and enter a project name. Your new project is built and after a few seconds begins running in the cloud.
+
+## Import an existing local project to the cloud
+
+Copy an existing local project, for example `myFirstNodeProject`, over to the remote Codewind deployment:
+
+1\. Select the remote deployment and click `Add Existing Project`:
+
+![Add existing project](./images/remotedeploy-projects-eclipse/eclipse_add-project.png){:width="366"}
+
+2\. Go to the folder that contains the local project files, click `Select a project from the workspace`, and select your project: 
+
+![Add to Codewind](./images/remotedeploy-projects-eclipse/eclipse_select-project.png){:width="654"}
+
+Codewind prompts you to confirm the project type, in this case, identified as `NodeJS`:
```

Comment:
Should probably match what is in the screen capture which is `Node.js` rather than `NodeJS`. The nodejs org site also refers to it as Node.js: https://nodejs.org/en/.